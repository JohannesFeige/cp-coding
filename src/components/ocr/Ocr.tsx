import React from 'react';
import BankOcr from './BankOcr';

const Ocr: React.FC = () => {
  return (
    <div>
      <h3>BankOCR</h3>
      <p>Scans account numbers from ASCII file.</p>
      <BankOcr />
    </div>
  );
};

export default Ocr;
