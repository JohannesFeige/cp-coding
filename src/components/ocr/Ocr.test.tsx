import React from 'react';
import { shallow } from 'enzyme';
import Ocr from './Ocr';

describe('Ocr', () => {
  it('should render', () => {
    const wrapper = shallow(<Ocr />);
    expect(wrapper).toMatchSnapshot();
  });
});
