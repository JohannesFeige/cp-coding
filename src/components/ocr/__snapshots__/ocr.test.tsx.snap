// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ocr should render 1`] = `
<div>
  <h3>
    BankOCR
  </h3>
  <p>
    Scans account numbers from ASCII file.
  </p>
  <BankOcr />
</div>
`;
